cmd/cloud.go

```diff
@@ -59,285 +59,286 @@ var (
 	showCloudLogs = true
 )
 
-//nolint:gochecknoglobals
-var cloudCmd = &cobra.Command{
-	Use:   "cloud",
-	Short: "Run a test on the cloud",
-	Long: `Run a test on the cloud.
+//nolint:funlen,gocognit,gocyclo
```

Comment:
It feels like we replaced one bad `nolint` with a few worse ones 😄

Let's work on these soon as well :)

Reply:
The only reason those weren't already there was that we haven't touched near the start of the inner function since we have golangci-lint :rofl: .

Reply:
Yeah, I know, but it reads like they're new issues, even though we're used to them, so it's good to see these `nolint`s explicitly.